src/test/javascript/spec/component/organization/organization-management.service.spec.ts

```diff
@@ -44,6 +44,13 @@ describe('Organization Service', () => {
         req.flush(JSON.stringify(elemDefault));
     });
 
+    it('should return an Organization with Users and Courses', async () => {
```

Comment:
This is `async`, so I expect we need to `tick()` after flushing the request? (Is the subscription executed without ticking?)

Reply:
We probably also want to use `fakeAsync`